workflows/mag.nf

```diff
@@ -667,8 +668,19 @@ workflow MAG {
             )
             ch_versions = ch_versions.mix(PROKKA.out.versions.first())
         }
+
+        /*
+        * DAS_Tool: binning refinement
+        */
+        if ( params.refine_bins_dastool && !params.skip_metabat2 && !params.skip_maxbin2 ) {
+            BINNING_REFINEMENT ( BINNING_PREPARATION.out.grouped_mappings, BINNING.out.bins )
+            ch_versions = ch_versions.mix(BINNING_REFINEMENT.out.versions)
+        }
```

Comment:
If I want to run DAS_Tool, don't I expect that this refined bins are actually going into Prokka, QUAST, GTDB, CAT, etc? I understand that DAS_Tool bins will not used at all, but instead the not-refined bins? I find that not optimal.

Reply:
Lets talk on slack about this, I was in two minds